interface/main/messages/messages.php

```diff
@@ -36,15 +36,13 @@
 require_once("$srcdir/log.inc");
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/gprelations.inc.php");
+
+use OpenEMR\Core\Header;
 ?>
+<!DOCTYPE html>
 <html>
 <head>
-
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-2-2/index.js"></script>
+    <?php Header::setupHeader(['dialog', 'textformat', 'jquery-ui']); ?>
```

Comment:
textformat and dialog are automatically included when run setupHeader
(so, only need to put jquery-ui here)

quick links:
https://github.com/openemr/openemr/blob/master/config/config.yaml#L72
https://github.com/openemr/openemr/blob/master/config/config.yaml#L76